protocol/scripts/deploys/sepolia/v2/post-deploy/999-transfer-ownership.ts

```diff
@@ -40,7 +40,6 @@ async function main() {
 
         await proposeCore(TEMPLE_V2_ADDRESSES, TEMPLE_V2_INSTANCES.RAMOS.TEMPLE_DAI.INSTANCE);
         await proposeCore(TEMPLE_V2_ADDRESSES, TEMPLE_V2_INSTANCES.RAMOS.TEMPLE_DAI.AURA_STAKING);
```

Comment:
Why was 
```
await proposeCore(TEMPLE_V2_ADDRESSES, TEMPLE_V2_INSTANCES.RAMOS.TEMPLE_DAI.INSTANCE);
```
removed? It should also change owner

Reply:
prob by mistake will revert

Reply:
sorry, just checked, it is there on line 52 I believe?

Reply:
on line 52? Hmm i don't see that, line 52 is referencing the TEMPLE_BASE_STRATEGY

Reply:
Ohh line numbers mixed up. I see it is on line 41 -- this was a duplicate